### PR TITLE
Adding Key A and Key B management

### PR DESCRIPTION
**Pull request by emanuelr93**
Refer to this issue: https://github.com/davetcc/tcMenuLib/issues/59

### CODE REVIEW

File: src/tcMenuKeyboard.cpp [hunk 1]
```diff
@@ -52,6 +52,41 @@ void MenuEditingKeyListener::keyPressed(char key, bool held) {
...
+		}
+		
+	}
+	else if (key == 'B') {
+		MenuItem* itm = menuMgr.getCurrentMenu();
+		bool haveSelected = false;
+		while (itm != NULL) {
+			if (itm->isActive()) {
+				itm->setActive(false);
+				itm->getNext()->setActive(true);
```
Comment (davetcc) [inline src/tcMenuKeyboard.cpp:79]: Please can you wrap this in a null check, the item will have a NULL next. IE itm->getNext() may be NULL.

File: src/tcMenuKeyboard.cpp [hunk 2]
```diff
@@ -52,6 +52,41 @@ void MenuEditingKeyListener::keyPressed(char key, bool held) {
 		// we are not editing, attempt to select an item using 0-9
 		menuMgr.valueChanged(fromKeyToIndex(key));
 	}
+	else if (key == 'A') {
+		MenuItem* itm = menuMgr.getCurrentMenu();
+		bool haveSelected = false;
+		while (itm != NULL) {
+			if (itm->getNext()->isActive()) {
+				itm->getNext()->setActive(false);
```
Comment (davetcc) [inline src/tcMenuKeyboard.cpp:60]: Please can you wrap this in a null check, the item will have a NULL next. IE itm->getNext() may be NULL. For example:
if (itm->getNext() && itm->getNext()->isActive()) {

**Comment by emanuelr93** [comment]
@davetcc Thank you for your feedback. Sorry but I'm not a programmer C. I fixed what you suggested. I hope it goes well.
Please let me know.

**Comment by davetcc** [comment]
> @davetcc Thank you for your feedback. Sorry but I'm not a programmer C. I fixed what you suggested. I hope it goes well.
> Please let me know.

No problem at all, that's how we learn, by trying things and asking people for feedback. Many thanks for submitting the change.

**Comment by emanuelr93** [comment]
> > @davetcc Thank you for your feedback. Sorry but I'm not a programmer C. I fixed what you suggested. I hope it goes well.
> > Please let me know.
> 
> No problem at all, that's how we learn, by trying things and asking people for feedback. Many thanks for submitting the change.

Many thanks to you for giving this wonderful library and the enteire ecosystem around it to the community.

**Pull request merged**